contract_mandate/models/contract.py

```diff
@@ -32,8 +32,10 @@ def _onchange_payment_mode_id(self):
             self.mandate_id = False
 
     @api.multi
-    def _prepare_invoice(self, date_ref=False):
-        invoice_vals = super(ContractContract, self)._prepare_invoice(date_ref)
+    def _prepare_invoice(self, date_invoice, journal=None):
+        invoice_vals = super(ContractContract, self)._prepare_invoice(
+            date_invoice, journal
```

Comment:
Please respect always keyword arguments:
```suggestion
            date_invoice, journal=journal
```